entrypoint.sh

```diff
@@ -3,10 +3,18 @@ set -euo pipefail
 
 STORAGE_SERVICE_ALIAS=${STORAGE_SERVICE_ALIAS:="s3"}
 STORAGE_SERVICE_URL=${STORAGE_SERVICE_URL:="https://s3.amazonaws.com"}
+SESSION_TOKEN=${SESSION_TOKEN:=""}
 MIRROR_SOURCE=${MIRROR_SOURCE:="."}
+REGION=${REGION:=""}
 
 # Set mc configuration
-mc alias set "$STORAGE_SERVICE_ALIAS" "$STORAGE_SERVICE_URL" "$ACCESS_KEY_ID" "$SECRET_ACCESS_KEY"
+if [ -z "$SESSION_TOKEN" ]; then
+  mc alias set "$STORAGE_SERVICE_ALIAS" "$STORAGE_SERVICE_URL" "$ACCESS_KEY_ID" "$SECRET_ACCESS_KEY"
+  # Execute mc mirror
+  mc mirror $* "$MIRROR_SOURCE" "$STORAGE_SERVICE_ALIAS/$MIRROR_TARGET"
+else
+  export MC_HOST_${STORAGE_SERVICE_ALIAS}=https://${ACCESS_KEY_ID}:${SECRET_ACCESS_KEY}:${SESSION_TOKEN}@s3.${REGION}.amazonaws.com
+  # Execute mc mirror
+  mc mirror $* "$MIRROR_SOURCE" "$STORAGE_SERVICE_ALIAS/$MIRROR_TARGET"
```

Comment:
These two lines are duplicated in both paths of the `if` block. I think you can just extract this out after the `if` block.

Reply:
Sure I'll remove the duplicate
I did this in a bit of a rush and that's why I opened the PR to check on my code.